app/models/feeds/apple_subscription.rb

```diff
@@ -77,4 +87,31 @@ def apple?
   def publish_to_apple?
     !!apple_config&.publish_to_apple?
   end
+
+  def default_feed_audio_format
+    af = podcast&.default_feed&.audio_format
+    standardize_audio_format(af) if af && af[:f] == "mp3"
+  end
+
+  def episode_audio_format
+    episodes = podcast&.episodes&.published&.includes(:contents)&.limit(10)
+    contents = (episodes || []).map { |e| e.contents.first }.compact
+    mp3_contents = contents.select { |c| c.audio? && c.mime_type == "audio/mpeg" }
+
+    if mp3_contents.any?
+      max_bitrate = mp3_contents.map { |c| c.bit_rate.to_i }.max
+      max_channels = mp3_contents.map { |c| c.channels.to_i }.max
+      max_sample = mp3_contents.map { |c| c.sample_rate.to_i }.max
```

Comment:
this seems right, better to use the max, i.e. better to upscale than downscale quality